src/Application/Adapters/Drupal/Preset.php

```diff
@@ -20,7 +20,7 @@ public static function getName(): string
     }
 
     /**
-     * {@inheritDoc}
+     * {@inheritdoc}
```

Comment:
Didn't we remove the inheritdocs tag in another commit? Do we want them or not?

Reply:
I find them not relevant.

Reply:
:sweat_smile: Good question. I just rewrite them here because [this rule](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.15|fixer:phpdoc_inline_tag) enforce write them in lowercase, but I can also remove them.

Reply:
Haha, Yeah just remove them :)